apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -278,7 +281,7 @@ function get_onboarding_url() {
 					</div>
 					<div class="wpcom-coming-soon-marketing-buttons">
 						<p><a class="button button-secondary" href="<?php echo esc_url( get_login_url() ); ?>"><?php esc_html_e( 'Log in', 'full-site-editing' ); ?></a></p>
-						<p><a class="button button-primary " href="<?php echo esc_url( get_onboarding_url() ); ?>"><?php esc_html_e( 'Start your website', 'full-site-editing' ); ?></a></p>
+						<p><a class="button button-primary has-background" href="<?php echo esc_url( get_onboarding_url() ); ?>"><?php esc_html_e( 'Start your website', 'full-site-editing' ); ?></a></p>
```

Comment:
Adding this class prevents some CSS rules (that were partially causing the issue) from being applied

![image](https://user-images.githubusercontent.com/1083581/114590007-85d8df00-9c88-11eb-9eed-4eb0651f8128.png)